internal/datasourcev2/polling_data_source.go

```diff
@@ -18,10 +18,47 @@ const (
 	pollingWillRetryMessage = "will retry at next scheduled poll interval"
 )
 
-// Requester allows PollingProcessor to delegate fetching data to another component.
```

Comment:
A bit of new binding code so that the requestor can pass back FDv2 related data.